MySQL.Data/tests/MySql.Data.Tests/TestBase.cs

```diff
@@ -333,6 +333,37 @@ public string GetMySqlServerIp(bool isIpV6 = false)
 
       return isIpV6 ? ipv6 : ipv4;
     }
+
+    // XXX This shouldn't be the final form - it probably shouldn't be placed
```

Comment:
This isn't very good; I'm fairly new to dotnet unit testing and NUnit, so I tried to be a little generic but not too much. In general I wasn't sure whether to make specific tests for `MySQLActivitySource`, since it seemed like it would be duplicating all the great tests that currently exist.

Something I was doubly unsure of: This isn't relevant for runtimes which don't have `MySQLActivitySource`.